label_studio/users/models.py

```diff
@@ -26,6 +27,10 @@
 class UserManager(BaseUserManager):
     use_in_migrations = True
 
+    def __init__(self, disable_is_deleted_filter=False, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.disable_is_deleted_filter = disable_is_deleted_filter
```

Comment:
please define and type this field on the UserManager class, like `use_in_migrations` above (which should really be `use_in_migrations: bool = True`)